test/functional/kernel/nested-loop/test-kernel-nested-loop.cpp.in

```diff
@@ -94,10 +94,12 @@ using CudaKernelNestedLoopExecPols = camp::list<
     NestedLoopData<DEVICE_DEPTH_2, RAJA::cuda_thread_x_loop, RAJA::cuda_thread_y_loop >,
     NestedLoopData<DEVICE_DEPTH_2, RAJA::seq_exec, RAJA::cuda_thread_x_loop >,
     NestedLoopData<DEVICE_DEPTH_2, RAJA::cuda_thread_x_loop, RAJA::seq_exec >,
+    NestedLoopData<DEVICE_DEPTH_2, RAJA::cuda_global_x_direct<32>, RAJA::cuda_global_y_loop<8> >,
```

Comment:
What is tested is fairly arbitrary right now, any ideas of how to make more general or parametrized kernel and launch tests?

Reply:
We should probably re-assess how we are doing this in the tests. I would prefer that it is less opaque.

Reply:
As separate discussion, do we like how these tests are generated with CMake configure files? I've sometimes found them difficult trace through since they aren't fully generated until compile time.

Reply:
There is a reason for that, google test generates a lot of template instantiations and that caused some compilers take huge amounts of time to compile those tests. So we split the tests up using the generated files to improve build times.